core/Request.php

```diff
@@ -244,6 +268,60 @@ public function post($index = false, $default = false)
         return $this->post[$index];
     }
 
+    public function delete($index = false, $default = false)
```

Comment:
Request->delete() doesn't seem right.

For that matter Request->get() really doesn't seem right either.  
$this->request->get(); doesn't seem intuitive.

I apologize for lacking a more technical response here.